pkg/standalone/standalone.go

```diff
@@ -112,6 +114,44 @@ func Init(runtimeVersion string, dockerNetwork string, installLocation string) e
 	return nil
 }
 
+// nolint:gosec
+// GetLatestRelease return the latest release version of dapr
+func GetLatestRelease(gitHubOrg, gitHubRepo string) (string, error) {
+	releaseURL := fmt.Sprintf("https://api.github.com/repos/%s/%s/releases", gitHubOrg, gitHubRepo)
+	resp, err := http.Get(releaseURL)
+	if err != nil {
+		return "", err
+	}
+	defer resp.Body.Close()
+
+	if resp.StatusCode != 200 {
+		return "", fmt.Errorf("%s - %s", releaseURL, resp.Status)
+	}
+
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return "", err
+	}
+
+	var githubRepoReleases []githubRepoReleaseItem
+	err = json.Unmarshal(body, &githubRepoReleases)
+	if err != nil {
+		return "", err
+	}
+
+	if len(githubRepoReleases) == 0 {
+		return "", fmt.Errorf("no releases")
+	}
+
+	for _, release := range githubRepoReleases {
+		if !strings.Contains(release.TagName, "-rc") {
+			return release.TagName, nil
+		}
+	}
+
+	return "", fmt.Errorf("no releases")
+}
+
```

Comment:
This one is shared in k8s and standalone modes. Then, I would move this function to https://github.com/dapr/cli/blob/master/pkg/version/version.go

Reply:
That's better, moved it.